nupic/algorithms/CLAClassifier.py

```diff
@@ -213,6 +213,31 @@ def __setstate__(self, state):
     self._version = BitHistory.__VERSION__
 
 
+  def write(self, proto):
+    proto.id = self._id
+
+    statsProto = proto.init("stats", len(self._stats))
+    for (bucketIdx, dutyCycle) in enumerate(self._stats):
+      statsProto[bucketIdx].index = bucketIdx
+      statsProto[bucketIdx].dutyCycle = dutyCycle
+
+    proto.lastTotalUpdate = self._lastTotalUpdate
+    proto.learnIteration = self._learnIteration
+
+
+  def read(self, proto):
```

Comment:
> No I don't quite follow

No problem.. I solved it. I had to call `classifier = cls()` instead of `classifier = object.__new__(cls)`. It seems that `__new__` doesn't call `__init__` and so many fields are not created.